app/views/posts/index.html.erb

```diff
@@ -1 +1,22 @@
-<p>List of all posts and comment of a user</p>
\ No newline at end of file
+<%= render 'shared/user', user: @user %>
+<% @posts.each do |post| %>
+  <div class="post-details">
+    <h3>Post:<%= link_to post.id, user_post_path(@user, post) %></h3>
+    <p><%= post.text %></p>
+     <div class="counters">
+        <p>Comments: <%= post.comments_counter %> Likes: <%= post.likes_counter %> </p>
+      </div>
+  </div>
+   <% if post.recent_comments.any? %>
+      <div class="user-comments">
+        <% post.recent_comments.each do |comment| %>
+          <div>
+            <p><strong><%= comment.author.name %>:</strong> Comment:<%= comment.id %></p>
+          </div>
```

Comment:
- [ ]  You did an amazing job, and Your app is working perfect, just one small thing you need to fix is that You should not display `comment.id`, but comment itself, which was supposed to be `comment.text.`  Please make sure that You follow the provided wireframe and display the comment right value  for your app to work perfect.